src/style.css

```diff
@@ -1 +1,26 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('./images/star-active.svg');
+}
```

Comment:
The way you're selecting child elements is correct, but ensure that the images used in the `background-image` property are optimized for performance.